scripts/healthcheck.ts

```diff
@@ -1,34 +1,255 @@
+      console.log(`${dir}/: ✓ Exists`);
+    } else {
+      console.log(`${dir}/: ✗ Missing`);
+    }
+  });
+  console.log('');
+
+  console.log('--- Log File Analysis ---');
   if (existsSync('error.log')) {
-    const errorLogContent = readFileSync('error.log', 'utf8').trim();
-    if (errorLogContent.length > 0) {
-      console.error(`Error Log (${errorLogContent.length} chars): ${errorLogContent}`);
-      process.exit(2);
+    try {
+      const errorLogContent = readFileSync('error.log', 'utf8').trim();
+      if (errorLogContent.length > 0) {
+        console.log(`Error Log (${errorLogContent.length} chars):`);
+        const lines = errorLogContent.split('\n');
+        if (lines.length > 20) {
+          console.log('... (showing last 20 lines) ...');
+          console.log(lines.slice(-20).join('\n'));
+        } else {
+          console.log(errorLogContent);
+        }
+        
+        if (errorLogContent.toLowerCase().includes('fatal') || 
+            errorLogContent.toLowerCase().includes('critical') ||
+            errorLogContent.toLowerCase().includes('traceback')) {
+          hasFatalError = true;
+        }
+      } else {
+        console.log('Error log exists but is empty');
+      }
+    } catch (error) {
+      console.log(`Failed to read error.log: ${error}`);
     }
+  } else {
+    console.log('No error.log file found');
+  }
+
+  if (existsSync('browser-error.log')) {
+    try {
+      const browserErrorContent = readFileSync('browser-error.log', 'utf8').trim();
+      if (browserErrorContent.length > 0) {
+        console.log(`Browser Error Log (${browserErrorContent.length} chars):`);
+        const lines = browserErrorContent.split('\n');
+        if (lines.length > 10) {
+          console.log('... (showing last 10 lines) ...');
+          console.log(lines.slice(-10).join('\n'));
+        } else {
+          console.log(browserErrorContent);
+        }
+      } else {
+        console.log('Browser error log exists but is empty');
+      }
+    } catch (error) {
+      console.log(`Failed to read browser-error.log: ${error}`);
+    }
+  } else {
+    console.log('No browser-error.log file found');
+  }
+
+  if (existsSync('agent.log')) {
+    try {
+      const agentLogContent = readFileSync('agent.log', 'utf8').trim();
+      if (agentLogContent.length > 0) {
+        console.log(`Agent Log (${agentLogContent.length} chars):`);
+        const lines = agentLogContent.split('\n');
+        if (lines.length > 10) {
+          console.log('... (showing last 10 lines) ...');
+          console.log(lines.slice(-10).join('\n'));
+        } else {
+          console.log(agentLogContent);
+        }
+      } else {
+        console.log('Agent log exists but is empty');
+      }
+    } catch (error) {
+      console.log(`Failed to read agent.log: ${error}`);
+    }
+  } else {
+    console.log('No agent.log file found');
+  }
+  console.log('');
+
+  console.log('--- Network Connectivity ---');
```

Comment:
Remove this section from console logging and add to the same ongoing file string